x-pack/solutions/security/plugins/security_solution/public/management/components/endpoint_responder/command_render_components/run_script_action.tsx

```diff
@@ -87,7 +120,9 @@ export const RunScriptActionResult = memo<
         agentId={command.commandDefinition?.meta?.endpointId}
         textSize="s"
         data-test-subj="console"
-        hideFile={true}
+        // Currently file is not supported for CrowdStrike
```

Comment:
Intentionally refering to CS, that's the only one we want to hide file for now. MDE should show file 👍